stdpopsim/slim_engine.py

```diff
@@ -288,15 +292,44 @@
 """
 
 
+def join_wrap(a, delim, wrap=80, newline="\n"):
```

Comment:
Can't we do this with [textwrap](https://docs.python.org/3.6/library/textwrap.html)?

Reply:
Oh, cool. I never knew that existed.